src/modules/shortcut_guide/shortcut_guide.cpp

```diff
@@ -52,6 +55,17 @@ namespace
         {
             return result;
         }
+
+        DWORD pid;
+        WCHAR exePath[MAX_PATH]{}; 
+        GetWindowThreadProcessId(active_window, &pid);

```

Comment:
We should always check the return value and handle the error case.

Reply:
Not really sure on what is returned in case of error: https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getwindowthreadprocessid#return-value

Reply:
As an alternative, initialize `pid` to zero and check is not zero after the call.